src/Microsoft.IdentityModel.Clients.ActiveDirectory/UserIdentifier.cs

```diff
@@ -61,7 +61,7 @@ public sealed class UserIdentifier
         private const string AnyUserId = "AnyUser";
 
         private static readonly UserIdentifier AnyUserSingleton =
-            new UserIdentifier(AnyUserId, UserIdentifierType.UniqueId);
+            new UserIdentifier(AnyUserId, UserIdentifierType.OptionalDisplayableId);
```

Comment:
this looks like a breaking change in behavior. can you confirm?

Reply:
as I see from the code - Nothing depends on AnyUser type, we only pass it to the broker,  Android brokers does not care about it, IOS broker  expect object id for uniqueId type

Reply:
yes,  but it is not supposed to be used outside of Adal Api